tools/bundle-size-tools/src/utilities/getChunkAndDependenciesSizes.ts

```diff
@@ -49,10 +50,10 @@ export function getChunkAndDependencySizes(stats: Stats.ToJsonOutput, chunkName:
   const dependencySizeInfo: ChunkSizeInfo[] = [];
 
   // To avoid duplicate work, keep track of all the dependencies we have already examined
-  const processedDependencies = new Set<number>();
+  const processedDependencies = new Set<number|string>();
 
   // Get the initial set of dependencies
-  const dependenciesToProcess = [...rootChunk.parents, ...rootChunk.siblings];
+  const dependenciesToProcess = [...rootChunk.parents??[], ...rootChunk.siblings??[]];
```

Comment:
Nit: The de facto convention is to put spaces around the '??' operator.

Reply:
I miss my auto format on save. Fixed.

Reply:
Our shared lint config should catch this... but it looks like it's not used in this project. I opened #10351 for that.